http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/AppService.scala

```diff
@@ -9,33 +9,33 @@ import org.broadinstitute.dsde.workbench.model.UserInfo
 trait AppService[F[_]] {
   def createApp(
     userInfo: UserInfo,
-    cloudContext: CloudContext,
+    cloudContext: CloudContext.Gcp,
```

Comment:
All the v1 app methods are GCP-specific. Rather than throw exceptions in each one if the cloudContext is Azure, I thought we could just constrain it at the type level.

Reply:
we were thinking using this for both v1 and v2 routes so that we can re-use code as much as possible..

Reply:
oh nvm, I see u're only changing the methods in this file that are specific to v1 routes....

Reply:
yea that makes sense....I'd like to populate workspaceId at some point in leo DB and migrate to use only v2 routes